From 242b315ee63010211f519b6dd87b020fb2582647 Mon Sep 17 00:00:00 2001
From: Howard Butler <hobu.inc@gmail.com>
Date: Mon, 27 Nov 2023 10:25:01 -0600
Subject: [PATCH] add OWNER_WRITE permissions to installs #34258

---
 CMakeLists.txt | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5bec0d7443..ef845879e6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -312,6 +312,7 @@ install(DIRECTORY ${PDAL_INCLUDE_DIR}/pdal/
     PERMISSIONS
         GROUP_EXECUTE WORLD_EXECUTE OWNER_EXECUTE
         GROUP_READ WORLD_READ OWNER_READ
+        OWNER_WRITE
 #    PATTERN "pdal/private" EXCLUDE
 #    PATTERN "pdal/io/private" EXCLUDE
 #    PATTERN "pdal/pdal/private" EXCLUDE
@@ -328,6 +329,7 @@ install(DIRECTORY ${PDAL_KERNELS_DIR}/
     PERMISSIONS
         GROUP_EXECUTE WORLD_EXECUTE OWNER_EXECUTE
         GROUP_READ WORLD_READ OWNER_READ
+        OWNER_WRITE
 #    PATTERN "private" EXCLUDE
 )
 install(DIRECTORY ${PDAL_IO_DIR}/
@@ -336,6 +338,7 @@ install(DIRECTORY ${PDAL_IO_DIR}/
     PERMISSIONS
         GROUP_EXECUTE WORLD_EXECUTE OWNER_EXECUTE
         GROUP_READ WORLD_READ OWNER_READ
+        OWNER_WRITE
 #    PATTERN "private" EXCLUDE
 )
 install(DIRECTORY ${PDAL_FILTERS_DIR}/
@@ -344,6 +347,7 @@ install(DIRECTORY ${PDAL_FILTERS_DIR}/
     PERMISSIONS
         GROUP_EXECUTE WORLD_EXECUTE OWNER_EXECUTE
         GROUP_READ WORLD_READ OWNER_READ
+        OWNER_WRITE
 #    PATTERN "private" EXCLUDE
 )
 
@@ -352,12 +356,14 @@ install(FILES ${DIMENSION_OUTFILE} ${pdal_features_hpp}/
     PERMISSIONS
         GROUP_EXECUTE WORLD_EXECUTE OWNER_EXECUTE
         GROUP_READ WORLD_READ OWNER_READ
+        OWNER_WRITE
 )
 install(FILES ${PDAL_CMAKE_DIR}/pluginmacros.cmake
     DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/PDAL
     PERMISSIONS
         GROUP_EXECUTE WORLD_EXECUTE OWNER_EXECUTE
         GROUP_READ WORLD_READ OWNER_READ
+        OWNER_WRITE
         )
 
 #
