diff -ruN a/setup.py b/setup.py
--- a/setup.py	2021-11-16 17:25:31.000000000 +0100
+++ b/setup.py	2022-01-16 18:51:43.553731798 +0100
@@ -48,7 +48,6 @@
     author=about["__author__"],
     author_email=about["__email__"],
     python_requires=">=3.6",
-    install_requires=["pyparsing>=2.0.2,!=3.0.5"],  # 2.0.2 + needed to avoid issue #91
     classifiers=[
         "Development Status :: 5 - Production/Stable",
         "Intended Audience :: Developers",
